src/types/LocalEnums.ts

```diff
@@ -13,6 +13,32 @@ export enum LocalAddressType {
   Billing = 'Billing',
 }
 
+export enum LocalPaymentRequestEventType {
```

Comment:
The property paymentAttempts already exists. It could be used instead of the events.